app/backend/src/couchers/phone/sms.py

```diff
@@ -18,5 +22,37 @@ def format_message(token):
 
 def send_sms(number, message):
     """Send SMS to a E.164 formatted phone number with leading +. Return "success" on
-    success, on failure return an error string for any other error."""
-    return errors.UNSUPPORTED_OPERATOR
+    success, "unsupported operator" on unsupported operator, and any other
+    string for any other error."""
+
+    assert len(message) <= 140, "message too long"
+
+    if not config["ENABLE_SMS"]:
+        logger.info(f"SMS not enabled, need to send to {number}: {message}")
+        return
+
+    sns = boto3.client("sns")
+    sender_id = config.get("SMS_SENDER_ID")
```

Comment:
```suggestion
    sender_id = config["SMS_SENDER_ID"]
```

Will give an easier to understand traceback, than passing None in the dict later on, in case one forgets to set this env variable.

Reply:
Thanks! Good point